test/lib/worker.test.js

```diff
@@ -482,8 +482,8 @@ describe('Worker library', () => {
         processExitTimeout: 1
       });
       yield worker.listen();
-      yield worker.close();
-      yield worker.wait('worker.closed');
+      yield worker.close(); // true by default
+      yield cb => setTimeout(cb, 1000);
```

Comment:
The test was better before, 1 second is very long for a unit test, and the test can now fail randomly on slow machines

Reply:
Yes but it doesn't work anymore... the event is triggered before the wait could catch it. I'm just following the same approach as the one in stakhanov : 
https://github.com/ChauffeurPrive/stakhanov/blob/master/test/lib/worker.test.js#L412-L414